primitives/avn-common/src/event_types.rs

```diff
@@ -208,21 +208,17 @@ impl AddedValidatorData {
 #[derive(Encode, Decode, Default, Clone, PartialEq, Debug, Eq, TypeInfo, MaxEncodedLen)]
 pub struct LiftedData {
     pub token_contract: H160,
-    pub sender_address: H160,
     pub receiver_address: H256,
     pub amount: u128,
     pub nonce: U256,
 }
 
 impl LiftedData {
     const TOPIC_CURRENCY_CONTRACT: usize = 1;
-    const TOPIC_INDEX_T1_ADDRESS: usize = 2;
     const TOPIC_INDEX_T2_ADDRESS: usize = 3;
```

Comment:
I think this needs to change as well, now that I go into the details. Now that we don't have the sender data in the event, we lose one of the topics, so what comes after it (the T2 address) moves up one position to be number 2.